src/Makefile.am

```diff
@@ -38,6 +38,7 @@ gnotime_SOURCES =     \
 	props-proj.c       \
 	props-task.c       \
 	query.c            \
+	select-list.c      \
```

Comment:
Please add the `select-list.c` to the `CMakeLists.txt` file in this directory too.